Avoid implicit function declarations and build failures with future
compilers.

Submitted upstream: <https://github.com/SawfishWM/rep-gtk/pull/3>

diff --git a/build-gtk.jl b/build-gtk.jl
index cf6d28a61bc011c6..917330a970b1d4de 100644
--- a/build-gtk.jl
+++ b/build-gtk.jl
@@ -508,6 +508,7 @@
       (@ "  static int done;\n  if (!done)\n    {\n")
       (@ "      done = 1;\n")
       (mapc (lambda (func)
+	      (@ "      extern void %s (void);\n" func)
 	      (@ "      %s ();\n" func)) other-inits)
       (when (or gtk-enums gtk-string-enums gtk-flags gtk-boxed gtk-objects)
 	(@ "      sgtk_register_type_infos (_type_infos);\n"))
diff --git a/rep-gtk.h b/rep-gtk.h
index 2a0026af3a595069..e4cf48a0cc402f17 100644
--- a/rep-gtk.h
+++ b/rep-gtk.h
@@ -341,6 +341,13 @@ void gtk_color_selection_set_color_interp (GtkColorSelection *sel, GdkColor *col
 GdkColor *gtk_color_selection_get_color_interp (GtkColorSelection *sel);
 char *gtk_color_button_get_color_interp (GtkColorButton *button);
 extern void gtk_widget_draw_interp (GtkWidget *widget);
+void gtk_status_icon_popup_menu(GtkStatusIcon *status_icon,
+				GtkMenu *menu,
+				guint button,
+				guint32 activate_time);
+gboolean gtk_status_icon_get_geometry_interp(GtkStatusIcon *status_icon,
+					     gint *x, gint *y,
+					     GtkOrientation *orientation);
 
 GtkTextIter *gtk_text_iter_new (void);
 repv sgtk_gtk_widget_get_allocation (GtkWidget *w);
